src/widgets/codes-lists/components/codes-lists-input-code.jsx

```diff
@@ -199,6 +208,13 @@ class CodesListInputCode extends ComponentWithValidation {
                       label={Dictionary.maxLength}
                       onEnter={this.addCodeIfIsValid1}
                     />
+                    <button
+                    className={`${CODE_INPUT_ACTIONS_CLASS}-cancel`}
+                    onClick={remove}
```

Comment:
You should add a aria-label attribute here in order to give some meaning to this button.